### PR TITLE
Review: LogReg and Scan

### PR DESCRIPTION
**Pull request by abergeron**

### CODE REVIEW

File: intro_theano/logistic_regression.ipynb [hunk 1]
```diff
@@ -153,7 +153,7 @@
    "cell_type": "code",
    "execution_count": null,
    "metadata": {
-    "collapsed": true
+    "collapsed": false
```
Comment (nouiz) [inline intro_theano/logistic_regression.ipynb:156]: do you really want this?

File: intro_theano/logistic_regression.ipynb [hunk 2]
```diff
@@ -413,6 +413,15 @@
...
    ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": []
```
Comment (nouiz) [inline intro_theano/logistic_regression.ipynb]: is this change expected?

File: intro_theano/logistic_regression.ipynb (hunk 1 above)
Comment (abergeron) [inline intro_theano/logistic_regression.ipynb:156]: Not really, but it shouldn't make a difference.

File: intro_theano/logistic_regression.ipynb (hunk 2 above)
Comment (abergeron) [inline intro_theano/logistic_regression.ipynb]: Yes and no.  This adds an empty cell at the end which is the result of doing CTRL-Enter on the last cell.  I tend to leave it in in the notebooks I make.

Comment (abergeron) [inline intro_theano/logistic_regression.ipynb]: I just removed the trailing cells.

**Pull request merged**